.github/workflows/ci.yml

```diff
@@ -0,0 +1,18 @@
+name: Continuous Integration
+
+
+on:
+  pull_request:
+    branches:
+      - main
+    paths-ignore:
+      - '**/README.md'
+      - '**/LICENSE'
+
+
+jobs:
+
+  build:
+    name: Build for PR
+    uses: ./.github/workflows/build.yml
+    # explicitly not passing secrets into the build
```

Comment:
double-checking: pull requests originating from outside the `Descent3` repository will never get access to secrets, no matter what - correct? ie, even if the PR modifies this file, or even if someone _merges_ a modification to this file?

_(is it even possible to submit a PR to yourself? suppose i could try that out, but kinda irrelevant either way)_

Reply:
I'll do an actual live test of this tomorrow.

Reply:
The live test was a success.  I confirmed GitHub's documentation to be correct.  A PR from a fork has _no_ access to the secrets in the base repository.